app/index.js

```diff
@@ -1437,6 +1437,21 @@ module.exports = JhipsterGenerator.extend({
             this.removefile('Cassandra-Dev.Dockerfile');
             this.removefile('Cassandra-Prod.Dockerfile');
             this.removefolder('docker/');
+
+            if (this.buildTool === 'gradle') {
```

Comment:
You can do away with the if checks as remove file method will check for file and delete if present otherwise it will just procees, so it could be just the methid calls